src/main/drivers/compass_hmc5883l.h

```diff
@@ -17,7 +17,7 @@
 
 #pragma once
 
-#include "drivers/io.h"
```

Comment:
Is there any advantage to removing directory names? With directory prefix, it is easier to find the file ...